src/arch/riscv64/CodeGen.zig

```diff
@@ -918,38 +1038,71 @@ pub fn addExtraAssumeCapacity(self: *Self, extra: anytype) u32 {
     return result;
 }
 
-fn gen(self: *Self) !void {
-    const mod = self.bin_file.comp.module.?;
-    const fn_info = mod.typeToFunc(self.fn_type).?;
+const required_features = [_]Target.riscv.Feature{
```

Comment:
Is it ever possible to not have this feature set for a RISCV target?

Reply:
It is, and in my current working branch, this `required_features` list includes vectors. This is because I have not done any work on emulating certain features like atomics, floats, vectors, or multiplication. That will come in due time, similar to the arbitrary float ints.